cmd/aries-mobile-sdk/.gitignore

```diff
@@ -0,0 +1,7 @@
+#
```

Comment:
i wonder if the “-sdk” suffix will cause confusion.

Reply:
Current 2 bindings doesn't have `-framework` or `-sdk` suffix (`aries-agent-rest` and `aries-js-worker`). So, I guess, just `aries-mobile` or `aries-agent-mobile` would be ok.

Reply:
i see what you mean @troyronda.
@rolsonquadras  i was thinking of `aries-mobile-agent`